published/052-2019-july-30.md

```diff
@@ -0,0 +1,59 @@
+## Welcome to the IPFS Weekly. 👋
+
+The [InterPlanetary File System (IPFS)](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identity. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open. Since that’s a pretty large scope, we track development across the ecosystem in this weekly dispatch.
+
+Looking to get involved? Click on some of the links below, see what we’re up to on [GitHub](https://github.com/ipfs), or join us on [IRC](https://riot.im/app/#/room/#ipfs:matrix.org).
+
+Want this update in your inbox? [Subscribe to our weekly newsletter!](https://tinyletter.com/ipfsnewsletter)
+
+Here are some of the highlights since the last IPFS Weekly.
+
+## The latest
+
+### 
+
+### 
+
+### 
+
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/articles/).* 
+
++ [Unbundling Vectors of Centralization in Web3](https://multicoin.capital/2019/07/25/unbundling-vectors-of-centralization-in-web3/), Multicoin Capital, 25 July 2019
++ Check out [Elk: The Dev Board for the Decentralized World](https://www.kickstarter.com/projects/233173198/elk-the-dev-board-for-the-decentralized-world), Kickstarter, 25 July 2019
++ [Report: A study of libp2p and ETH2](https://discuss.libp2p.io/t/report-a-study-of-libp2p-and-eth2/229), Raúl Kripalani, 24 July 2019
++ [Re-decentralizing the Web](https://medium.com/@hajanetworks/re-decentralizing-the-web-54678a1e4848), Haja Networks, 24 July 2019
++ [Smart Contract Based IPFS Storage for DAOs](https://medium.com/open-work-labs/smart-contract-based-ipfs-storage-for-daos-39c145f3042d), Open Work Labs, 24 July 2019
++ [ARK Core: Path And Vision To V3](https://blog.ark.io/ark-core-path-and-vision-to-v3-7a8bc3338d5a), ARK.io, 24 July 2019
++ [10+ Resources Every IPFS Developer Should Know About](https://medium.com/swlh/10-resources-to-get-started-with-ipfs-5f429dc8a841), The Startup, 23 July 2019
++ [IPFS Camp, new features, July 2019](https://peergos.org/blog#ipfs_camp_new_features_july_2019_), Peergos, 22 July 2019
```

Comment:
Could we please change the link title here to "Peergos at IPFS Camp, July 2019" please? Thank you.